features/enums.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Weekday(models.IntegerChoices):
+    """Weekday enum with values specified by
```

Comment:
Shouldn't the docstring be in the format:
```python
"""
Weekday enum with values specified by
[iso8601](https://en.wikipedia.org/wiki/ISO_8601).
"""
```
🙂 ?

Reply:
Maybe, maybe, not sure what is the most appropriate format for multi-line docstring summaries. [Some](https://google.github.io/styleguide/pyguide.html#381-docstrings) [sources](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) seem to agree, that with the format I'm using, the summary line itself should fit into one physical line.

Personally I've used to the following format:

```
class A:
    """A short one line summary

    One blank line + the rest of this docstring with overall description.
    """
```

Maybe I'll just refactor this to conform with this formatting?

Reply:
ah, it was the summary line. then I can agree with the format 🙂

Reply:
but OTOH it's not a one-line summary if it does not fit on one line 🤔 🤔 🤔